Twilio-org/phonebank
# POST /users/:id/callback endpoint

# Description
- Set up status callback endpoint under /users/:id/callback to accept POSTS
   - determined that call status doesn't matter because [Twilio will post to callback](https://www.twilio.com/docs/api/twiml/twilio_request#asynchronous) once the call is ended
   - wrote a service called `clearUserCallSID` and reused it in the `clearUserCallSIDField` controller as well as the `volunteerCallback` controller.
## notes
I notice that there are similar controller functions for clearing the call sid...could we reuse controllers functions for different routes? Feedback appreciated 😄 

### test plan
- Go to routes/user.js to see the route
- Go to services/user.js to see the new service
- Go to controllers/users.js to see the new controller
- `npm run test` to see tests pass
